src/Table/web/Table.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '../../theme';
+import THead from './THead';
+import TBody from './TBody';
+import TH from './TH';
+import TR from './TR';
+import TD from './TD';
+
+const Table = styled.table`
+  border-collapse: collapse;
+  background-color: ${(props) => props.color ? props.theme.color[props.color] : ''};
+`;
+
+Table.propTypes = {
+  color: PropTypes.oneOf(Object.keys(theme.color)),
+};
+
+Table.defaultProps = {
+  color: 'white',
+};
+
+Table.THead = THead;
```

Comment:
Table.Head

Reply:
However, that doesn't mean that we're not okay with coming up with a new set of names that's at least close enough to the original html semantics.